### DISCUSSION TITLE
Add join by proxy

### DISCUSSION THREAD

**Discussion by BigboiKilljoy** [YimMenu/YimMenu#1438] [Ideas]
You should add an option to the menu so that when you add someone to the database it chooses another random person from the session and also adds them to the database, this way when any modders spoof them selfs as offline and cant be joined you can still join them by joining someone else thats in their session

**Comment by Yimura** [comment]
And how exactly would we find out other people in their session if they have RID join protection enabled?

Using a proxy RID of a player that is only going to be temporarily be sharing the session is a waste of DB space.
If you really wish to keep stalking them I suggest you keep the "proxy" person in your clipboard and join them instead.
Once the person you're stalking switches session with RID join protection enabled you won't be able to follow them through the proxy either.

And if you know their friend doesn't have the RID join protection, just add them to the DB, there's a button under the info accordion to add them to the DB.

PS.:
I'm not sure RID joining even works after the recent update.

**Reply by BigboiKilljoy** [reply]
Im not trying to stalk people im just trying to join any modders that kick me and kick them back, problem is a cant join them if they're spoofing. I also wont remember any other players that might have been in the session because i wouldn't be paying attention to that 24/7
